app/controllers/conferences_controller.rb

```diff
@@ -11,6 +11,11 @@ def index
 
   def show; end
 
+  def current
+    current = Conference.where('start_date <= ? AND end_date >= ?', Date.current, Date.current).first
```

Comment:
if conference.start_date is tomorrow and one conference is live then the `current` variable will have the live conference and the next day both the conferences will be there in `current`. So the next day the home page will be the index action